tests/data_replicator.rs

```diff
@@ -164,7 +166,7 @@ pub fn crdt_retransmit() {
     assert!(done);
     let mut b = Blob::default();
     b.meta.size = 10;
-    Crdt::retransmit(&c1, &Arc::new(RwLock::new(b)), &tn1).unwrap();
+    Crdt::retransmit(&c1, &Arc::new(RwLock::new(b)), &tn1)?;
```

Comment:
if this function fails, you wont see it in the stackframe.  unless the Err itself is generated with a line number.